src/server.ts

```diff
@@ -0,0 +1,59 @@
+import * as express from "express";
+import * as morgan from "morgan";
+import * as bodyParser from "body-parser";
+import * as helmet from "helmet";
+import config from "./config";
+import { log } from "./utils/logger/logger";
+import { SeverityLevel } from "./utils/logger/severityLevel";
+import addHeaders from "./utils/addHeaders";
+import * as errorhandlers from "./utils/erros/errorHandlers";
+import appRouter from "./router";
+
+export class Server {
+  private static _instance: Server;
+  public app: express.Application;
+
+  private constructor() {
+    this.app = express();
```

Comment:
please add middleware for validating the token with passport and passport-jwt. 
if you need one, you can ask me.
otherwise other users can send request to your service without any authentication with postman.